### DISCUSSION TITLE
Problem with merge repo from git when not merged with master branch

### DISCUSSION THREAD

**Discussion by ewa-radomska-codeclusive** [codemagic-ci-cd/codemagic-docs#1552] [Q&A] [answered]
Hello, from 3 weeks I had weird problem when building flutter app (triggered by pull requests). When branch is not updated with master, I always see error like this:

```
== Merging with remote source/some-source commit hash: d75d31f66ea544bc2495bd300d91 ==
> /usr/bin/git merge source/some-source d75d31f66ea544bc2495bd300d91

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <builder@z-builder-642ecabe4b9bb42c673.c.woven-voyage-217607.internal>) not allowed

```

If I do "git merge master" before push my branch, everything seems to be ok. I don't have any .git/config files in repository. Seems like VM hasn't proper configuration? But why only for branches which are not updated with master?
Did anyone face similar problem? I will be grateful for any advice :)

**Comment by himesh-cm** [comment]
Hi @ewa-radomska-codeclusive, we are looking into this and i will get back to you.

**Reply by himesh-cm** [reply, marked as answer]
Can you please try again, let us know if the issue still persists.

**Reply by ewa-radomska-codeclusive** [reply]
Oh, yes! Working now! :) thanks a lot for your help :)
